precompiles/outposts/stride/setup_test.go

```diff
@@ -34,23 +35,24 @@ func TestPrecompileTestSuite(t *testing.T) {
 
 func (s *PrecompileTestSuite) SetupTest() {
```

Comment:
it's not possible for me to comment in that line but in line 20 of this file there is: `var _ *PrecompileTestSuite` which we should remove I think